<?xml version="1.0"?>

<!-- map.launch -->
<launch>
    <!-- Follower -->
    <node name="path_follower" pkg="igvc_navigation" type="path_follower" output="screen" >
        <!-- Maxmimum velocity allowed before stopping motors -->
        <param name="maximum_vel" type="double" value="2"/>
        <param name="axle_length" type="double" value="0.52"/>
        <!-- Distance along the path to generate smooth control law for  -->
        <param name="lookahead_dist" type="double" value="1.3"/>
        <!-- granularity with which to visualize planned trajectories -->
        <param name="granularity" type="int" value="100"/>
        <!-- How far away robot should be from the waypoint before it stops -->
        <param name="stop_dist" type="double" value="0.7"/>
        <!-- Smooth Control Parameters -->
        <param name="target_v" type="double" value="0.4"/>
        <param name="k1" type="double" value="1.8"/>
        <param name="k2" type="double" value="5"/>
        <param name="simulation_horizon" type="double" value="5"/>
        <param name="debug" type="bool" value="true"/>
    </node>

</launch>
